.env.local

```diff
@@ -1,2 +1,5 @@
 PUBLIC_BACKEND_ENDPOINT=http://localhost:3333/
-PUBLIC_BACKEND_WS_ENDPOINT=ws://localhost:3333/
\ No newline at end of file
+PUBLIC_BACKEND_WS_ENDPOINT=ws://localhost:3333/
+
+# copy to pkg/web .env file
+PUBLIC_FARO_WEBSDK_URL=your faro web URL
```

Comment:
We can now delete this, it's not needed anymore.

Reply:
Absolutely right, it's now gone!